superset/assets/src/visualizations/MapBox/MapBox.css

```diff
@@ -0,0 +1,16 @@
+.mapbox div.widget .slice_container {
```

Comment:
could you tell if the `.widget` styles still apply without dashboard v1?

Reply:
I could not find any element on both explore and dashboard so I delete these css.